api/onboardingpb/json/onboardingpb.json

```diff
@@ -0,0 +1,204 @@
+{
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "schemes": [
+    "https",
+    "http"
+  ],
+  "swagger": "2.0",
+  "info": {
+    "title": "PMM Platform API",
+    "version": "v1"
+  },
+  "paths": {
+    "/v1/onboarding/tips/{tipId}/complete": {
+      "post": {
+        "description": "Method marks user tip is done. Only works for user tip, not for system tips.",
+        "tags": [
+          "TipService"
+        ],
+        "summary": "Complete user tip",
+        "operationId": "CompleteUserTip",
+        "parameters": [
+          {
+            "type": "integer",
+            "format": "int32",
+            "description": "The ID of the tip to retrieve.",
+            "name": "tipId",
+            "in": "path",
+            "required": true
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "description": "Request message for CompleteUserTip method.",
+              "type": "object",
+              "properties": {
+                "userId": {
+                  "description": "user id which completes user tip.",
+                  "type": "integer",
+                  "format": "int32"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "description": "Response message for CompleteUserTip method.",
+              "type": "object"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-order": 0
+                },
+                "details": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "@type": {
+                        "type": "string",
+                        "x-order": 0
+                      }
+                    },
+                    "additionalProperties": false
+                  },
+                  "x-order": 2
+                },
+                "message": {
+                  "type": "string",
+                  "x-order": 1
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/v1/onboarding/{userId}": {
```

Comment:
not good from API design

`/v1/onboarding/{userId}` - is not subresource, so userId should not be a part for path.

but `userId` should not be present because we take it from contex

Reply:
yes, it's not good design, but we don't have ability to read http headers in our code when we use GRPC over HTTP (like it works now), because this code is autogenerated.

What can we do it's to put the userId to request body and change method to POST, but not sure that it will look better.